v2/tools/asoctl/pkg/importing/import_azure_resource.go

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT license.
+ */
+
+package importing
+
+import (
+	"context"
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore/cloud"
+	"github.com/Azure/azure-service-operator/v2/tools/asoctl/internal/importing"
+	"github.com/devigned/pub/pkg/xcobra"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+	"k8s.io/klog/v2"
+	"os"
+)
+
+func newImportAzureResourceCommand() *cobra.Command {
+	var output *string
+
+	cmd := &cobra.Command{
+		Use:   "azure-resource <ARM/ID/of/resource>",
+		Short: "imports an ARM resource as a CR",
+		Args:  cobra.ExactArgs(1),
+		Run: xcobra.RunWithCtx(func(ctx context.Context, cmd *cobra.Command, args []string) error {
+			armID := args[0]
+			return importAzureResource(ctx, armID, output)
+		}),
+	}
+
+	output = cmd.Flags().StringP(
```

Comment:
Logging always seems to go to stdout, so it may not always be straight-forward to simply redirect the output to a file; having a `--output` option seems like a good way to make simple things simple.